taskcluster/app_services_taskgraph/target_tasks.py

```diff
@@ -10,8 +10,9 @@
 def target_tasks_pr_skip(full_task_graph, parameters, graph_config):
     return []
 
+@_target_task('nightly')
```

Comment:
I've never tried setting this decorator to the same function. I wonder if it works as intended 🤔